binary_transparency/firmware/docs/design/README.md

```diff
@@ -131,27 +131,62 @@ following diagram:
 
 ![overview diagram](./overview.svg)
 
-For clarity, the mapping of actors to claimant model roles are listed explicitly here:
+For clarity, the mapping of actors to the claimant model roles, along with
+software provided by the demo used to fulfil those roles, are listed explicitly
+here:
 
 **Firmware vendor**:
+
+Uses the [`publisher`](/binary_transparency/firmware/cmd/publisher) to publish
+firmware metadata & images to the log, and create an "OTA" update bundle.
+
 * Claimant<sup>FIRMWARE</sup>
```

Comment:
We've missed `Verifier(FIRMWARE)` for the "I should check my key material is only being applied when I expected" use case

Reply:
Good spot, thanks!
PTAL.